src/config/rt/serve.rs

```diff
@@ -250,7 +250,21 @@ fn absolute_path_if_some(
     file_description: &str,
 ) -> anyhow::Result<Option<PathBuf>, anyhow::Error> {
     match maybe_path {
-        Some(path) => Ok(Some(absolute_path(path, file_description)?)),
+        Some(path) => {
+            let path = if path.display().to_string().contains('~') {
```

Comment:
I think we can go with `to_string_lossy`, which should prevent allocation and might be a bit shorter.